src/Components/Analyzers/test/AnalyzerTestBase.cs

```diff
@@ -49,19 +48,8 @@ public Task<Compilation> CreateCompilationAsync(string source)
 
         private static string GetProjectDirectory()
```

Comment:
nit for readability is it worth considering just removing this helper and inlining AppContext.BaseDirectory everywhere now?